server/routerlicious/kubernetes/routerlicious/values.yaml

```diff
@@ -171,3 +171,6 @@ error:
 lumberjack:
   options:
     enableGlobalTelemetryContext: true
+
+config:
+  configDumpEnabled: true
```

Comment:
Maybe this can also be set to false by default to match what we have in the config.json files?